datasets/oneoff/21-1.cgel

```diff
@@ -9,10 +9,10 @@
         :Head (Nom
             :Head (N :t "court")))
     :Head (VP
-        :Head (V_aux :t "may")
+        :Head (V_aux :t "may" :l "may" :xpos "MD")
```

Comment:
We use `:l` only for lemmas that differ from the `:t` token

Reply:
oops... thanks